src/s11/handlers/delete_session_handler.c

```diff
@@ -52,6 +51,12 @@ struct MsgBuffer*  dsReqMsgBuf_p = NULL;
 static int
 delete_session_processing(struct DS_Q_msg *ds_msg)
 {
+	struct MsgBuffer*  dsReqMsgBuf_p = createMsgBuffer(1024);
```

Comment:
can we use a single macro for the buffer size?

Reply:
Yes, done.